Mathlib/Analysis/Calculus/EDeriv/Basic.lean

```diff
@@ -0,0 +1,176 @@
+/-
+Copyright (c) 2025 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov, Sam Lindauer
+-/
+import Mathlib.Analysis.NormedSpace.Alternating.Uncurry.Fin
+import Mathlib.Analysis.Calculus.FDeriv.Symmetric
+import Mathlib.Analysis.Calculus.FDeriv.CompCLM
+
+/-!
+# Exterior derivative of a differential form
```

Comment:
As a differential geometer, when I hear "differential form", I think "on a manifold".
Is that a me-problem, or can you tweak at least the module doc to make this clearer?